README.md

```diff
@@ -178,405 +128,133 @@ After installing the library you will need to require it in your test file (see
 
-### Example of usage of `waitForUrlToContainString`
+That method fills a file input field with a specified file as soon as the file input field is present in the DOM.
+[Example](docs/EXAMPLES.md#fillinputfieldwithfilewhenpresent)
 
-```js
-const protractorHelper = require("protractor-helper");
+### `clearFieldWhenVisible`
 
-describe("foo", () => {
-    it("bar", () => {
-        browser.get("https://example.com");
+That method clears a text input field as soon as such field is visible.
 
-        const goToContactPageButton = element(by.className("contact-button"));
+### `clearFieldWhenVisibleAndFillItWithText`
 
-        goToContactPageButton.click();
+That method clears a text input field as soon as such field is visible, and then it fills it with a text.
+[Example](docs/EXAMPLES.md#clearfieldwhenvisibleandfillitwithtext)
 
-        protractorHelper.waitForUrlToContainString("contact", "URL does not contains the string 'contact'", 3000);
+### `tapWhenTappable`
 
-        // ...
-    });
-});
-```
+That method performs a tap action on a clickable/tappable HTML element as soon is it is clickable/tappable. This method is used when performing web mobile testing in mobile emulators, for example.
+[Example](docs/EXAMPLES.md#tapwhentappable)
 
-### Example of usage of `waitForUrlNotToContainString`
+### `waitForTextToBePresentInElement`
 
-```js
-const protractorHelper = require("protractor-helper");
+That method waits for a specific text to be present in a specific HTML element.
+[Example](docs/EXAMPLES.md#waitfortexttobepresentinelement)
 
-describe("foo", () => {
-    it("bar", () => {
-        browser.get("https://example.com");
+### `waitForTextNotToBePresentInElement`
 
-        protractorHelper.waitForUrlNotToContainString("foobarbaz", "URL contains the string 'foobarbaz'", 3000);
+That method is the opposite of the previous one, so, it waits for a specific text not to be present in a specific HTML element.
+[Example](docs/EXAMPLES.md#waitfortextnottobepresentinelement)
 
-        // ...
-    });
-});
-```
-
-### Example of usage of `fillFieldWithTextWhenVisibleAndPressEnter`
-
-```js
-const protractorHelper = require("protractor-helper");
+### `waitForUrlToBeEqualToExpectedUrl`
 
-describe("foo", () => {
-    it("bar", () => {
-        browser.get("https://example.com");
+That method waits for the URL to be equal to an expected URL. Such method is useful when you want to continue performing actions on elements only when in the correct URL.
+[Example](docs/EXAMPLES.md#waitforurltobeequaltoexpectedurl)
 
-        const textField = element(by.css("input.some-text-field"));
+### `waitForUrlNotToBeEqualToExpectedUrl`
 
-        protractorHelper.fillFieldWithTextWhenVisibleAndPressEnter(textField, "some text", "textField is not visible", 3000);
+That method waits for the URL not to be equal to an expected URL. Such method is useful when you want to continue performing actions on elements only when not in a specific URL.
```

Comment:
Same as others.

Reply:
Commit made because this review: ce8ef21